modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -822,6 +832,93 @@ public void tearDown() throws Exception {
 			_objectEntryLocalService, _objectRelationshipLocalService);
 	}
 
+	@FeatureFlag("LPD-47858")
+	@Test
+	public void testAddObjectEntryCategoriesLazyReference() throws Exception {
```

Comment:
Thanks @AliciaGarciaGarcia. 

Can you just rename it to `testAddObjectEntryWithTaxonomyCategoryBriefs` to follow the existing pattern in this class?

Reply:
@AliciaGarciaGarcia for other references that we had to apply the lazy reference approach we are using the following pattern on Objects side:

`testAdd**FooEntity**WithMissing**BarEntity**Reference`

so please rename to `testAddObjectEntryWithMissingTaxonomyCategoryBriefReference`.

Thanks!

cc: @carolmariaabb